### PR TITLE
Fixing testcase

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @rishubhjain please review

File: tendrl/commons/tests/test_init.py [hunk 1]
```diff
@@ -73,7 +73,8 @@ def test_list_modules_in_package_path():
         ('platform', 'tendrl.commons.objects.platform'),
         ('service', 'tendrl.commons.objects.service'),
         ('tendrl_context', 'tendrl.commons.objects.tendrl_context'),
-        ('block_device', 'tendrl.commons.objects.block_device')]
+        ('block_device', 'tendrl.commons.objects.block_device'),
+        ('virtual_disk', 'tendrl.commons.objects.virtual_disk')]
```
Comment (rishubhjain) [inline tendrl/commons/tests/test_init.py]: I have already this change in -> https://github.com/Tendrl/commons/pull/559/files#diff-42379248bd3053683b4604f479e96dc2R78

Comment (GowthamShanmugam) [inline tendrl/commons/tests/test_init.py]: @rishubhjain ok, i am closing this PR

File: tendrl/commons/tests/utils/test_import_utils.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import pytest
 from tendrl.commons.utils import import_utils
+from tendrl.commons.tests.fixtures.config import Config
```
Comment (rishubhjain) [inline tendrl/commons/tests/utils/test_import_utils.py]: This file has been modified in the PR I have submitted, https://github.com/Tendrl/commons/pull/559/files#diff-a0746c6470fba44b0b9bf957a268985d

File: tendrl/commons/tests/objects/node_context/test_node_context_init.py [hunk 3]
```diff
@@ -126,18 +127,20 @@ def test_create_node_id(patch_write,patch_read, patch_client):
     NS.config["data"] = maps.NamedDict()
     NS.config.data['tags'] = "test"
     node_context=NodeContext(machine_id="Test_Machine_id", node_id="Test_Node_id", fqdn="Test_fqdn")
-    f = open('temp.txt','wb+')
+    f = tempfile.TemporaryFile()
```
Comment (rishubhjain) [inline tendrl/commons/tests/objects/node_context/test_node_context_init.py]: great, should have used this instead 👍

**Pull request closed without merging**